src/components/TranslatedError.js

```diff
@@ -18,12 +19,26 @@ export default function TranslatedError({ error, field = "title" }: Props) {
     return null;
   }
   // $FlowFixMe
-  const arg: Object = { message: error.message, ...error };
+  const arg: Object = { message: error.message, returnObjects: true, ...error };
```

Comment:
I'm not sure what does `returnObjects` ?

Reply:
by default , returnObjects= false.
the transition accepts only string. This was the old behaviour. 
Now, I have android/ios, nano s/x for different translation, so the translation is an object (an object contains keys like "ios", "android"....) You can refer to my "common.json " to understand how it works.
Thanks